test/system/require_login/users_test.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+require 'supports/login_assert_helper'
+
+class UsersLoginTest < ApplicationSystemTestCase
+  include LoginAssertHelper
+  test 'cannot access users list without login' do
+    assert_login_required('/users')
```

Comment:
```suggestion
    assert_login_required '/users'
```

複雑な場合以外はかっこはなくていいかもです〜

Reply:
修正しました！